python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -163,12 +166,12 @@ class wPerpTableConfig(Config):
     config.measure.raColumn = "coord_ra_new"
     """
 
-    bright_rmag_cut = Field(
-        doc="Bright magnitude limit to select", dtype=float, default=17.0
-    )
-    faint_rmag_cut = Field(
-        doc="Faint magnitude limit to select", dtype=float, default=23.0
-    )
+    #bright_rmag_cut = Field(
+    #    doc="Bright magnitude limit to select", dtype=float, default=17.0
+    #)
+    #faint_rmag_cut = Field(
+    #    doc="Faint magnitude limit to select", dtype=float, default=23.0
+    #)
```

Comment:
Why is this commented out? It might have been better to rebase some of these commits so that I'm not reviewing things that have been changed.

Reply:
Again, these mag cuts aren't in the PR'ed version of this ticket branch.

Reply:
A pull request includes all the commits on the pull request. It can be helpful to tell a reviewer if you will be squashing commits before merging and they should only look at the end state.

Reply:
I typically assume that a reviewer should only look at the end state, so it never occurred to me that anyone would look at prior commits. I’ll make sure to be more explicit about this in the future.

Reply:
I would not feel like I had done my due dilligence if I did not look at every commit included on a PR. It might have been better to rebase before putting it up for review, it is both confusing and time consuming to review parts of a PR that are not supposed to be there.